### PR TITLE
🌐 Add Spanish translation of Deployment - About HTTPS

### PR DESCRIPTION
**Pull request by pedabraham**
Hello 👋!,
This Pull Request includes the Spanish translation of **deployment/https.md**

##### This pull request continues with the Spanish translation of #1218

### CODE REVIEW

File: docs/es/docs/deployment/https.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Acerca de HTTPS
+
+Es bastante fácil asumir que HTTPS es algo que simplemente está "activado" o no.
+
+Pero en la realidad es algo mucho más complicado que eso.
+
+!!! tip
+    Si estas en un apuro o simplemente no te importa, continua con la siguientes secciones para ver una guía paso a paso para configurar todo con técnicas diferentes.
```
Comment (eavv) [inline docs/es/docs/deployment/https.md:8]:
```suggestion
    Si estás en un apuro o simplemente no te importa, continua con las siguientes secciones para ver una guía paso a paso para configurar todo con técnicas diferentes.
```

File: docs/es/docs/deployment/https.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    * Eso es una capa debajo de HTTP.
+    * Entonces, el manejo de certificado y encriptación es hecho antes de HTTP.
+* TCP no conoce de "dominios". Solo de direcciones IP.
+    * La información sobre el dominio específicamente requerido va en los datos de HTTP.
+* Los certificados HTTPS "certifican" un cierto dominio, pero el protocolo y la encriptación suceden en el nivel TCP, antes de conocer con que dominio se está lidiando.
+* Por defecto, eso diría que solamente puedes tener un certificado HTTPS por dirección IP.
+    * No importa que tan grande sea tu servidor, o que tan pequeñas puedan ser las aplicaciones que tienes en él.
+    * Sin embargo, existe una solución a esto.
+* Existe una extensión al protocolo TLS (quien maneja la encriptación a nivel TCP, antes de HTTP) llamado <a href="https://en.wikipedia.org/wiki/Server_Name_Indication" class="external-link" target="_blank"><abbr title="Server Name Indication">SNI</abbr></a>.
+    * Esta extensión SNI permite que un servidor particular (con una sola dirección IP) pueda tener varios certificados HTTPS y servir multiples dominios/aplicaciones HTTPS.
```
Comment (eavv) [inline docs/es/docs/deployment/https.md:29]:
```suggestion
    * Esta extensión SNI permite que un servidor particular (con una sola dirección IP) pueda tener varios certificados HTTPS y servir múltiples dominios/aplicaciones HTTPS.
```

File: docs/es/docs/deployment/https.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    * Para que esto funcione, un solo componente (programa) que este corriendo en el servidor, escuchando en la dirección IP publica, tenga todos los certificados HTTPS en el servidor.
```
Comment (eavv) [inline docs/es/docs/deployment/https.md:30]:
```suggestion
    * Para que esto funcione, un solo componente (programa) que esté corriendo en el servidor, escuchando en la dirección IP pública, tenga todos los certificados HTTPS en el servidor.
```

File: docs/es/docs/deployment/https.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+* Después de obtener una conexión segura, la comunicación del protocolo sigue siendo HTTP.
+    * El contenido es encriptado, incluso cuando está siendo enviado con el protocolo HTTP.
+
+Es una practica común tener un servidor programa/HTTP corriendo en el servidor (la maquina, el huésped, etc.) y manejando todas las partes de HTTPS: enviando la desencriptada petición HTTP a la aplicación HTTP actual que esta corriendo en el mismo servidor (la aplicación con **FastAPI**, en este caso), tome la respuesta HTTP de la aplicación,  la encripte usando el certificado apropiado y la envié de vuelta al cliente usando HTTPS. Este servidor es comúnmente llamado un <a href="https://en.wikipedia.org/wiki/TLS_termination_proxy" class="external-link" target="_blank">TLS Termination Proxy</a>.
```
Comment (eavv) [inline docs/es/docs/deployment/https.md:34]:
```suggestion
Es una práctica común tener un servidor programa/HTTP corriendo en el servidor (la máquina, el huésped, etc.) y manejando todas las partes de HTTPS: enviando la petición HTTP desencriptada a la aplicación HTTP actual que está corriendo en el mismo servidor (la aplicación con **FastAPI**, en este caso), tome la respuesta HTTP de la aplicación,  la encripte usando el certificado apropiado y la envié de vuelta al cliente usando HTTPS. Este servidor es comúnmente llamado un <a href="https://en.wikipedia.org/wiki/TLS_termination_proxy" class="external-link" target="_blank">TLS Termination Proxy</a>.
```

**Review by eavv — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**